install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -319,6 +319,16 @@
         "latestImageLayers": [
           "{{.JetBrainsPluginLatestImage}}",
           "{{.JetBrainsLauncherImage}}"
+        ],
+        "versions": [
```

Comment:
the script also ensures that the version specified on the top level of the ide option object is present in the versions array, so running it put at least the current versions in for rider and clion.

Reply:
They are working, we were just missing them?